pkgs/by-name/co/commitlint-rs/package.nix

```
@@ -0,0 +1,30 @@
+{
+  fetchFromGitHub,
+  lib,
+  nix-update-script,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "commitlint-rs";
+  version = "0.1.11";
+
+  src = fetchFromGitHub {
+    owner = "KeisukeYamashita";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-FrYXEh75H0u1rE1YNDL/B1gMYMG43jPDJGUMv9y5/3g=";
+  };
+  cargoSha256 = "sha256-W6HkLCUoylgQQc2fFprmJeLH8KtpVUD4+BXWbNECVZ4=";
+
+  passthru.updateScript = nix-update-script {};
+
+  meta = with lib; {
+    description = "Lint commit messages with conventional commit messages";
+    homepage = "https://keisukeyamashita.github.io/commitlint-rs";
+    changelog = "https://github.com/KeisukeYamashita/commitlint-rs/releases/tag/v${version}";
+    license = with licenses; [mit asl20];
+    mainProgram = "commitlint";
+    platforms = platforms.linux;
+    maintainers = [maintainers.croissong];
```

```suggestion
    maintainers = with maintainers; [ croissong getchoo ];
```
i'd love to help out here :)

---

gladly :)